system-tests/lib/dep-installer/index.ts

```diff
@@ -295,6 +295,7 @@ export async function scaffoldCommonNodeModules () {
     'lodash',
     'proxyquire',
     'react',
+    'react-dom',
```

Comment:
If we use `resolve/alias` to resolve the proper react/react-dom packages, do we need this change?

Reply:
I think if we have `react` here, we should include `react-dom` too - only caching one could lead to pretty confusing and hard to debug errors. Or, we should remove both. I saw `react` was here by itself, so I added `react-dom`.

Reply:
For consistency that makes sense. Ideally I think we should remove both but I imagine doing so would cause other tests to fail